packages/fake-driver/lib/driver.js

```diff
@@ -5,6 +5,12 @@ import {FakeApp} from './fake-app';
 import commands from './commands';
 
 class FakeDriver extends BaseDriver {
+  executeOverloads = {
```

Comment:
would this require breaking changes in existing drivers?

Reply:
no, existing drivers already implement something like this on their own and could continue to do so. but we could refactor them to take advantage of this feature and remove some code.